workdir/copy/link/run.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh
+
+plat=""
+arch=""
+fs=""
+net=0
+kvm=0
+extra_boot_args=""
+
+prepare_initrd()
+{
+    cd fs0
```

Comment:
```suggestion
    mkdir fs0
    cd fs0
```
Should we create the directory first?

Reply:
No, it's expected to already be there, part of the application.